src/common/utils/currencyConvert.ts

```diff
@@ -3,22 +3,18 @@
  */
 import type { CURRENCIES } from "../constants";
 
-const satsToFiat = async (amount: number | string, rate: number) => {
-  return Number(amount) * rate;
-};
-
-const satoshisToFiat = async ({
+const satoshisToFiat = ({
```

Comment:
As said below, this function is not used elsewhere
you may delete it as it does only `Number(amountInSats) * rate` 
see suggestion code [here](https://github.com/getAlby/lightning-browser-extension/pull/1643/files#r1000299863)